sources/packages/backend/apps/db-migrations/src/sql/DisbursementReceipts/Add-file-date-and-sequence-number-cols.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE
+  sims.disbursement_receipts
+ADD
+  COLUMN file_date DATE NOT NULL;
+
+COMMENT ON COLUMN sims.disbursement_receipts.total_entitled_grant_amount IS 'This reflects the date when the NSLSC file was produced.';
+
+ALTER TABLE
+  sims.disbursement_receipts
+ADD
+  COLUMN sequence_number NUMERIC(5) NOT NULL;
```

Comment:
In my understanding `sequence_number` will be an integer and can be declared as `INT`. Does it make sense?